log/logger.go

```diff
@@ -0,0 +1,96 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/CMSgov/bcda-ssas-app/ssas/constants"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/sirupsen/logrus"
+)
+
+// Logger provides a structured logger for this service
+var Logger logrus.FieldLogger
+
+// Event contains the superset of fields that may be included in Logger statements
+type Event struct {
+	UserID     string
+	ClientID   string
+	Elapsed    time.Duration
+	Help       string
+	Op         string
+	TokenID    string
+	TrackingID string
+}
+
+func init() {
```

Comment:
[Is there a duplicate of this init, event, and logger in ssas/logger](https://github.com/CMSgov/bcda-ssas-app/pull/151/files#r1443390402)?

Reply:
@laurenkrugen-navapbc Yes but it's still in use in other places of the codebase. I was planning on removing it in a following PR where I also refactor the use cases. I wanted this PR to be solely focused on adding the new context logger to keep it small. Does that sound alright to you?

Reply:
If we keep this in, can we get confirmation of a couple of things?
1. Are the writes to the file on `line 39` safe to do concurrently?
2. Are there any issues when running this locally / is the startup fine with having two of the same `init()`?

I don't anticipate issue with #2, but `init()` behavior is interesting and just want to be sure.

Reply:
For #1, it looks like writing to the same file is fine to do concurrently, based on [this piece in the documentation](https://pkg.go.dev/github.com/sirupsen/logrus#section-readme:~:text=logrus.RegisterExitHandler(handler)%0A...-,Thread%20safety,-By%20default%2C%20Logger) but  we should confirm the dupe `init()` behavior won't cause any other issues by running locally and also doing a quick check of the logs in dev once it's merged.

Reply:
Started app locally and was able to make successful calls to app using postman